Autopilot/Src/Callbacks.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Interchip_A.h"
+#include "imu.hpp"
+#include "spi.h"
+
+void HAL_SPI_TxRxCpltCallback(SPI_HandleTypeDef *hspi)
+{
+    if(hspi == &hspi1) {
```

Comment:
This is the right idea. Not hacky, it's good.

For reference, the really proper solution is to have each module pass the callback module a function pointer to call. That way, the callback module doesn't need to #include anyone and also that callback prototypes don't need to be exposed to the higher level. but that's of very low value and this solution is almost just as good.